### PR TITLE
Add changelog entry for dropping activesupport dependency

### PR DESCRIPTION
**Pull request by azgul**
Missed adding it in https://github.com/paper-trail-gem/paper_trail-sinatra/pull/11

### CODE REVIEW

**Comment by azgul** [comment]
A colleague of mine pointed out that I'd never added it. Somehow I had the changes staged and never committed/pushed them. Sorry.

Can I do anything to get this released?

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Added
 
 Fixed
 
-- None
+- Drop `activesupport` dependency. Implicitly introduces `activerecord` 6.0 support
```
Comment (jaredbeck) [inline CHANGELOG.md]:
```suggestion
- [#11](https://github.com/paper-trail-gem/paper_trail-sinatra/pull/11) - Drop 
  `activesupport` dependency, thus allowing `activerecord` 6.0
```

**Review by jaredbeck — CHANGES_REQUESTED** [review]

**Comment by jaredbeck** [comment]
@jkowens Jordan, can you cut a release please?

**Comment by jkowens** [comment]
@jaredbeck I don't have permission to push to rubygems.

**Comment by jaredbeck** [comment]
> I don't have permission to push to rubygems.

Oh, sorry. Can you please confirm that your rubygems account has MFA turned on, and then I'll add you (your public GH email, `jordan@copyandcode.com`) as a "gem owner".

**Comment by jkowens** [comment]
I have 2FA enabled, but I just updated my public GH email to jkowens@gmail.com

**Comment by jaredbeck** [comment]
```
gem owner paper_trail-sinatra --add 'jkowens@gmail.com'
Owner added successfully.
```

Thanks Jordan! Let me know if you have any questions about doing a release. Usually it involves updating:

1. `lib/paper_trail/sinatra/version.rb`,
1. the changelog, 
1. sometimes the readme

**Comment by jkowens** [comment]
v0.7.0 has been released!

**Pull request merged**